src/core/email_service.py

```
@@ -0,0 +1,374 @@
+    settings = settings or get_email_settings()
+
+    smtp_host = str(settings.get("SMTP服务器", "") or "").strip()
+    smtp_port = int(settings.get("SMTP端口", 465) or 465)
+    use_ssl = bool(settings.get("启用SSL", True))
+    sender = str(settings.get("发件邮箱", "") or "").strip()
+    password = str(settings.get("授权码", "") or "").strip()
+    timeout = int(settings.get("连接超时秒数", 30) or 30)
+
+    if not smtp_host or not sender or not password:
+        raise ValueError(
+            "邮件配置不完整，请在“设置 → 邮件发送配置”中填写 SMTP服务器、发件邮箱和授权码。"
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)